app/styleguide/wskComponentHandler.js

```diff
@@ -151,5 +150,16 @@ var componentHandler = (function() {
 window.addEventListener('load', function() {
   'use strict';
 
-  componentHandler.upgradeAllRegistered();
+  /**
+   * Performs a "Cutting the mustard" test. If the browser supports the features
+   * tested, adds a wsk-js class to the <html> element. It then upgrades all WSK
+   * components requiring JavaScript.
+   */
+  if ('classList' in document.createElement('div') && 'querySelector' in document && 'addEventListener' in window && Array.prototype.forEach) {
+    document.documentElement.className += 'wsk-js';
```

Comment:
Why using `.className += 'wsk-js'` if the browser supports classList? This way of adding classes might result in trouble later:

![screenshot 2015-01-26 20 24 18](https://cloud.githubusercontent.com/assets/6025224/5906626/7578c8b0-a599-11e4-99b8-50c15855d8a6.png)

Or at least add a space in the string you're appending, so it won't cause trouble.